doc/INSTALL.md

```diff
@@ -206,3 +207,123 @@ Alternatively, the kpatch and kpatch-build scripts can be run directly from the
 git tree.
 
 
+### OpenEuler
+
+*NOTE: You'll need about 15GB of free disk space for the kpatch-build cache in
+`~/.kpatch` and for ccache.*
+
+Install the dependencies for compiling kpatch and running kpatch-build:
+
+```bash
+source test/integration/lib.sh
+# Will request root privileges
+kpatch_dependencies
+```
+
+Before running kpatch-build, two more things need to be checked:
+-------
+1. Ensure current kernel compiled with *CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY* set
+
+    openEuler has two strategies to apply kernel live patches and it is decided at compile time.
+
+    When CONFIG_LIVEPATCH_STOP_MACHINE_CONSISTENCY set, openEuler uses its own strategy.
+
+    When CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY set, openEuler uses the conventional strategy.
+
+    Only one config option can take effect at the same time.
+    Differences between two strategies will not be discussed here, but you can easily guess them from config names.
+
+    Check whether your current kernel compiled with *CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY*
+    ```bash
+    grep -rn "CONFIG_LIVEPATCH_PER_TASK_CONSISTENCY" /boot/config-$(uname -r)
```

Comment:
nit: maybe grep `-rn` is muscle memory :) but strictly speaking, I think recursive search and line numbers are needed for this check

Reply:
yeah, it looks better now, already fix it.